### PR TITLE
docs: Add example Envoy escape hatch configs

### PR DESCRIPTION
**Pull request by blake**
Add example escape hatch configurations for all supported override types.

### CODE REVIEW

File: website/content/docs/connect/proxies/envoy.mdx [hunk 1]
```diff
@@ -471,33 +475,174 @@ definition](/docs/connect/registration/service-registration) or
   that will be appended to the array of [static
   clusters](https://www.envoyproxy.io/docs/envoy/v1.17.2/api-v3/config/bootstrap/v3/bootstrap.proto#envoy-v3-api-field-config-bootstrap-v3-bootstrap-staticresources-clusters)
   in the bootstrap config. This allows adding custom clusters for tracing sinks
```
Comment (trujillo-adam) [inline website/content/docs/connect/proxies/envoy.mdx:477]:
```suggestion
  in the bootstrap config. This enables you to add custom clusters for tracing sinks,
```

File: website/content/docs/connect/proxies/envoy.mdx [hunk 2]
```diff
@@ -471,33 +475,174 @@ definition](/docs/connect/registration/service-registration) or
...
-  for example. For a single cluster just encode a single object, for multiple,
-  they should be comma separated with no trailing comma suitable for
-  interpolating directly into a JSON array inside the braces.
+  for example. For a single cluster just, encode a single object. For multiple
```
Comment (trujillo-adam) [inline website/content/docs/connect/proxies/envoy.mdx:478]:
```suggestion
  for example. Encode a single object for a single cluster. For multiple
```
I'm a little unclear, is this what we mean?

File: website/content/docs/connect/proxies/envoy.mdx [hunk 3]
```diff
@@ -471,33 +475,174 @@ definition](/docs/connect/registration/service-registration) or
...
+  clusters, the objects should be comma separated with no trailing comma. The
```
Comment (trujillo-adam) [inline website/content/docs/connect/proxies/envoy.mdx:479]:
```suggestion
  clusters, specify objects in a comma-separated list with no trailing comma. The
```

**Review by trujillo-adam — COMMENTED** [review]
Thanks for adding the examples -- left a few suggestions.

File: website/content/docs/connect/proxies/envoy.mdx (hunk 2 above)
Comment (blake) [reply website/content/docs/connect/proxies/envoy.mdx:478]:
```suggestion
  for example. In order to configure a single cluster, specify a single JSON object with the cluster details. For multiple
```

Comment (blake) [reply website/content/docs/connect/proxies/envoy.mdx:478]: This is closer to what this sentence was trying to convey.

**Review by trujillo-adam — APPROVED** [review]
👍🏽

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/657655.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 54119f3225af89b3449764c91033beee71c55a9f onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 54119f3225af89b3449764c91033beee71c55a9f onto `release/1.12.x` succeeded!

**Pull request merged**